src/components/search/search-box.js

```diff
@@ -149,47 +131,13 @@ class SearchBox extends React.PureComponent {
             <div>
               {!props.resultsOnly && (
                 <React.Fragment>
-                  <label className="cursor-pointer" {...getLabelProps()}>
-                    <div
-                      className={classnames(
-                        'absolute flex-parent flex-parent--center-cross flex-parent--center-main',
-                        {
-                          'w60 h60': this.state.useModal,
-                          'w36 h36': !this.state.useModal
-                        }
-                      )}
-                    >
-                      <svg
-                        className={classnames('icon color-gray', {
-                          'w24 h24': this.state.useModal,
-                          'w18 h18': !this.state.useModal
-                        })}
-                      >
-                        <title>Search</title>
-                        <use xlinkHref="#icon-search" />
-                      </svg>
-                    </div>
-                    {this.props.isLoading && (
-                      <div
-                        className="loading loading--s absolute bg-white"
-                        style={{
-                          top: this.state.useModal ? '21px' : '10px',
-                          right: '26px',
-                          zIndex: 5
-                        }}
-                      />
-                    )}
-                  </label>
-                  <input
-                    ref={(input) => {
-                      this.docsSeachInput = input;
-                    }}
-                    placeholder={this.props.placeholder}
-                    className={classnames('input bg-white', {
-                      'px60 h60 txt-l': this.state.useModal,
-                      'px36 h36': !this.state.useModal
-                    })}
-                    {...getInputProps({
+                  <SearchInput
```

Comment:
💡 Created SearchInput and SearchButton components (at bottom of this file) so the SearchFacade could use them, too.